website/docs/docs/cloud/visual-editor-interface.md

```diff
@@ -0,0 +1,79 @@
+--- 
+title: "Visual editor interface" 
+id: visual-editor-interface      
+sidebar_label: "Visual editor interface" 
+description: "The visual editor interface contains a node toolbar, nodes, and canvas to help you create dbt models through a seamless drag-and-drop experience in dbt Cloud." 
+
+---
+
+# Visual editor interface <Lifecycle status='beta'/> 
+
+:::tip Beta feature
+The visual editor provides users with a seamless and visual, drag-and-drop experience inside dbt Cloud. It's available in private beta for [dbt Cloud Enterprise accounts](https://www.getdbt.com/pricing). 
+
+To join the private beta, [register your interest](https://docs.google.com/forms/d/e/1FAIpQLScPjRGyrtgfmdY919Pf3kgqI5E95xxPXz-8JoVruw-L9jVtxg/viewform) or reach out to your account team to begin this process.
+:::
+
```

Comment:
suggest moving this callout to line 18 instead (after the first paragraph on the page). per my previous SEO comment

Reply:
addressed here: https://github.com/dbt-labs/docs.getdbt.com/pull/5603#discussion_r1627538188